core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -121,6 +124,26 @@ public void testClusterTests() {
         }
     }
 
+    @ClusterTests({
+        @ClusterTest(clusterType = Type.ZK, disksPerBroker = 1),
```

Comment:
We can't use `disksPerBroker = 1` since it is equal to the default value.

Reply:
Thanks for finding this. Remove it.